src/index.js

```diff
@@ -0,0 +1,195 @@
+      } else if (like.innerText === '1 likes') {
+        like.innerText = '1 like';
+      }
+    });
+  });
+};
+// likes
+const tvId = 'MjgCDPvMKfBMbwFq4McF';
+
+const fetchLikeApi = async () => {
+  const getLikeResult = await fetch(`https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${tvId}/likes`).then((res) => res.json());
+  return getLikeResult;
+};
+
+const submitNewLike = async (id) => {
+  await fetch(`https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${tvId}/likes`, {
+    method: 'POST',
+    body: JSON.stringify({
+      item_id: id,
+    }),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  });
+};
+// popup
+
+const postComments = async (id, name, com) => {
+  await fetch('https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/4va6c4ouZmpzSETsANV3/comments', {
+    method: 'POST',
+    body: JSON.stringify({
+      item_id: id,
+      username: name,
+      comment: com,
+    }),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  });
+};
+
+const getComments = async (id) => {
+  const response = await fetch(`https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/4va6c4ouZmpzSETsANV3/comments?item_id=${id}`).then((res) => res.json());
+  return response;
+};
+
+document.addEventListener('click', async (e) => {
+  if (!e.target.matches('.commentBtn')) {
+    return;
+  }
+  const { id } = e.target;
+  const comList = await getComments(id);
+  const MoveInfo = await getData(id);
+  await display(MoveInfo, id, comList);
+  const container = document.querySelector('.modal-container');
+  container.style.display = 'none';
+
+  displayCom(comList);
+});
+
+// class Popup {
+const getData = async (id) => {
+  const movieInfo = await fetch(`https://api.tvmaze.com/shows/${id}`).then((result) => result.json());
+  return movieInfo;
+};
+
+const countComments = (number) => number.length;
+
+const display = async (movieInfo, id, comList) => {
+  const movie = await movieInfo;
+  const array = await comList;
+  const popup = document.querySelector('.popup');
+  popup.innerHTML = `
+      <div class="description">
+      <button type="button" class="closeBtn">X</button>
+        <div>
+          <a href="${movie.network.officialSite}"><img src="${movie.image.medium}" class = "image" id = "movie-img" alt=""></a>
+          <div class = "summary-tag">
+          <h1>${movie.name}</h1>
+          ${movie.summary}
+        </div>
+          
+            <div class = "form" >
+              <label for="fname">Name:</label><br>
+              <input type="text" id="fname" name="fname" placeholder = "Enter your name"><br><br>
+              <label for="comment">Comment:</label><br>
+              <textarea name="comment" id="comment" cols="20" rows="3" placeholder = "Enter comment"></textarea>
+              <button id = ${id} class= "submit" type="submit"> Submit</button>
+            </div>
```

Comment:
- [ ] Let's try to follow given mockup by adding title on comment add form and put comments before the form.

your design
![Screenshot from 2022-09-11 00-04-26](https://user-images.githubusercontent.com/65860678/189503465-4f0d1f7c-528f-4299-9ebd-e61313ed79bf.png)
Given design

![given](https://user-images.githubusercontent.com/65860678/189503519-d22026bd-5014-4edb-b944-6ca79cd2b643.png)